config/zones/DE.yaml

```diff
@@ -124,13 +124,19 @@ emissionFactors:
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 1087.81
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 931.5
     gas:
       - datetime: '2021-01-01'
         source: EU-ETS, ENTSO-E 2021
         value: 472.8075167
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 451.55
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 583.64
```

Comment:
Why did this increase? Can we double check it?

Reply:
We are using additional manual plant mappings that @w-flo put together here https://github.com/w-flo/eu-emission-factors/blob/main/data/2023/manual_matches.csv that seems to account for most changes as we are including more data.

Reply:
I was unable to recreate the existing 2021 and 2022 numbers when re-running the old notebook, so I did not feel confident updating past numbers at first (as I couldn't show why a certain number had changed). It might be worth doing, but I'd like to ensure we can clearly document all changes and I think it might be worth attempting to do it somewhat iteratively :)

Reply:
Makes sense.

I think once we are better able to explain how and why numbers are changing we can re run all the old data on a rolling basis whenever we update the emission factors. But let's just focus on 2023 for now :)